web_qr/README.rst

```diff
@@ -0,0 +1,92 @@
+==============
+Web QR Manager
+==============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/11.0/web_qr
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-web_qr
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to print QR in better structure than the standard odoo.
+
+The original image looks like the following
+
+.. figure:: https://raw.githubusercontent.com/OCA/hr/11.0/web_qr/static/description/old_qr.png
+   :alt: Original QR
+   :width: 100 px
+
+With the new generator, it looks like:
+
+.. figure:: https://raw.githubusercontent.com/OCA/hr/11.0/web_qr/static/description/new_qr.png
+   :alt: New QR
+   :width: 100 px
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+When creating new reports, you should use a path like the following::
+
+    <img t-att-src="'/report/qr/?value=%s&amp;error_correction=%s' % ('HELLO WORLD!', 3)" style="width:100;height:100"/>
```

Comment:
I would add some explanation on the possible parameters, as explained here: https://github.com/lincolnloop/python-qrcode/blob/master/README.rst

The `error_correction` parameter controls the error correction used for the QR Code. The following four constants are made available:
error_correction = 1: About 7% or less errors can be corrected.
error_correction = 0: About 15% or less errors can be corrected.
error_correction = 3: About 25% or less errors can be corrected.
error_correction = 2: About 30% or less errors can be corrected.

The `box_size` parameter controls how many pixels each "box" of the QR code is. The default is 3.
The `border` parameter controls how many boxes thick the border should be (the default is 4, which is the minimum according to the specs).

The `version` parameter is an integer from 1 to 40 that controls the size of the QR Code (the smallest, version 1, is a 21x21 matrix). Set to None and use the fit parameter when making the code to determine this automatically.

`fill_color` and `back_color` can change the background and the painting color of the QR, when using the default image factory.